calico_versioned_docs/version-3.24/operations/troubleshoot/commands.mdx

```diff
@@ -237,8 +237,8 @@ items:
 - apiVersion: operator.tigera.io/v1
   kind: Installation
   metadata:
-    - apiVersion: operator.tigera.io/v1
- spec:
+    name: my-installation
```

Comment:
What's this change about?

Reply:
Reverted the change. Fixed the yaml with an indentation change.